__tests__/intergration.tests.js

```diff
@@ -285,3 +289,56 @@ describe("/api/articles/:article_id/comments", () => {
       });
   });
 });
+describe("/api/articles/:article_id", () => {
+  test("PATCH: 202 updates the votes on an article by the article_id then returns the updated article with no other properties changed", () => {
+    return request(app)
+      .patch("/api/articles/1")
+      .send({ inc_votes: 5555 })
+      .expect(202)
+      .then((response) => {
+        expect(response.body.updatedArticle[0].votes).toBe(5655);
+        expect(response.body.updatedArticle[0].article_id).toBe(1);
+        expect(response.body.updatedArticle[0].author).toBe("butter_bridge");
+        expect(response.body.updatedArticle[0].title).toBe(
+          "Living in the shadow of a great man"
+        );
+        expect(response.body.updatedArticle[0].body).toBe(
+          "I find this existence challenging"
+        );
+        expect(response.body.updatedArticle[0].topic).toBe("mitch");
+        expect(typeof response.body.updatedArticle[0].created_at).toBe(
+          "string"
+        );
+        expect(typeof response.body.updatedArticle[0].article_img_url).toBe(
+          "string"
+        );
+      });
+  });
+  test("should not change any properties of any other articles", () => {
+    return request(app)
+      .patch("/api/articles/1")
+      .send({ inc_votes: 5555 })
+      .expect(202)
+      .then((response) => {
+        expect(response.body.updatedArticle.length).toBe(1);
+      });
+  });
+  test("should return 400 when given an invalid article_id", () => {
+    return request(app)
+      .patch("/api/articles/banana")
+      .send({ inc_votes: 5555 })
+      .expect(400)
+      .then((response) => {
+        expect(response.body.message).toBe("Bad Request");
+      });
+  });
+  test("should return 404 when given avalid but nonexistent article_id", () => {
+    return request(app)
+      .patch("/api/articles/999")
+      .send({ inc_votes: 5555 })
+      .expect(404)
+      .then((response) => {
+        expect(response.body.message).toBe("Not Found");
+      });
+  });
```

Comment:
Nice tests so far 🧪 
Could add one more test to check what happens when the object on the request body has no `inc_votes` property